taskcluster/ci/release-notify-testrail/kind.yml

```diff
@@ -0,0 +1,72 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.notify:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    description: Sends Slack message to release testers that Testrail Milestone was created.
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 120
+        env:
+            TESTRAIL_PROJECT_ID: '59' # Fenix Browser
+            TESTRAIL_TEST_SUITE_ID: '49319' # Test Automation Release Milestone
+    run:
+        use-caches: false
+        using: run-commands
+        secrets:
+            - name: project/mobile/ci/testrail
+              key: testrailCredentials
+              path: .testrail_credentials.json
+              json: true
+    routes:
+        - notify.slack-channel.G016BC5FUHJ.on-failed
+    scopes:
+        - queue:route:notify.slack-channel.G016BC5FUHJ # notify mobile-alerts-sandbox on failure
+        - notify:slack-channel:G016BC5FUHJ
+        - queue:route:notify.slack-channel.C02KDDS9QM9 # notify mobile-testeng on success
+        - notify:slack-channel:C02KDDS9QM9
+
+tasks:
+    create-milestone-focus:
+        dependencies:
+            ui-test-apk: ui-test-apk-focus-arm-beta
+        description: Create Testrail Milestone for Focus
+        run-on-tasks-for: [github-push]
+        run-on-git-branches: [releases_v]
+        run:
+            pre-commands:
+                # get-secrets is called from '..' directory so we need to cd into any directory to make it work
```

Comment:
Doh, that's annoying. Would it be simple to fix that rather than add this workaround?

Reply:
I a _little_ bit of time looking at it and I think the pathing was changed for all (most/some ?) of the `transform scripts` because of the mono-repo, and I was too afraid there was something else it was necessary for somewhere that I'm not accounting for.

I'm too afraid to modify any of the transforms yet since they are applied everwhere, or at least in some ways, I'm not 100% confident about.

It was definitely more of a skills/confidence gap, but might be simple to fix. The risk just seemed too high to me.